db/re/item_db_equip.yml

```
@@ -98688,9 +98686,7 @@ Body:
     Script: |
       bonus bCritical,5;
       .@r = getrefine();
-      if (.@r>=2) {
-         bonus bCritical,2;
-      }
+      bonus bCritical,.@r/2;
```

Delete the previous crit 5
```suggestion
      bonus bCritical,5+.@r/2;
```

---

Thanks